services/frontend-service/src/ui/components/dropdown/dropdown.tsx

```diff
@@ -13,11 +13,13 @@ You should have received a copy of the MIT License
 along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
 
 Copyright 2023 freiheit.com*/
-import { Checkbox, FormControl, InputLabel, MenuItem, Select } from '@material-ui/core';
+import { FormControl, InputLabel, MenuItem, Select } from '@material-ui/core';
 import classNames from 'classnames';
 import { useCallback, useRef } from 'react';
 import { useTeamNames } from '../../utils/store';
 import { useSearchParams } from 'react-router-dom';
+import { Checkbox as NewCheckBox } from '../dropdown/checkbox';
```

Comment:
This will be changed once we got rid of material-ui completely?

Reply:
fixed it